controllers/update_status.go

```diff
@@ -135,6 +134,7 @@ func (s UpdateStatus) Reconcile(r *FoundationDBClusterReconciler, context ctx.Co
 	if err != nil {
 		return false, err
 	}
+	removeDuplicateConditions(status)
```

Comment:
Do you know why these duplicates appear? From the update function we actually shouldn't have any duplicates?

Reply:
I don't, but I think it's valid to ensure explicitly that we are maintaining a contract of one entry per condition type.